src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -1110,46 +1067,31 @@ internal static StackValue[] GenerateRangeByAmount(decimal start, long amount, d
 
             if (svStart.IsNumeric)
             {
-                range = GenerateNumericRange(
-                    svStart,
-                    svEnd,
-                    svStep,
-                    svOp,
-                    hasStep,
-                    hasAmountOp,
-                    runtimeCore);
+                range = GenerateNumericRange(svStart, svEnd, svStep, svOp, hasStep, hasAmountOp, runtimeCore);
             }
             else
             {
                 if (svStart.IsString && !hasAmountOp)
                 {
-                    range = GenerateAlphabetRange(
-                    svStart,
-                    svEnd,
-                    svStep,                   
-                    runtimeCore);
+                    range = GenerateAlphabetRange(svStart, svEnd, svStep, runtimeCore);
                 }
                 else if (svStart.IsString && hasAmountOp)
                 {
-                    range = GenerateAlphabetSequence(
-                        svStart,
-                        svEnd,
-                        svStep,
-                        svOp,                        
-                        runtimeCore);
+                    range = GenerateAlphabetSequence(svStart, svEnd, svStep, svOp, runtimeCore);
                 }
             }
 
             return range == null ? StackValue.Null : runtimeCore.RuntimeMemory.Heap.AllocateArray(range);
         }
 
-        private static StackValue[] GenerateNumericRange(StackValue svStart,
-                                                         StackValue svEnd,
-                                                         StackValue svStep,
-                                                         StackValue svOp,
-                                                         bool hasStep,
-                                                         bool hasAmountOp,
-                                                         RuntimeCore runtimeCore)
+        private static StackValue[] GenerateNumericRange(
+            StackValue svStart,
+            StackValue svEnd,
+            StackValue svStep,
+            StackValue svOp,
+            bool hasStep,
+            bool hasAmountOp,
+            RuntimeCore runtimeCore)
```

Comment:
I'm wondering if there are test cases to test this method with all combinations (various `if-else` code paths) of range expression. Do you think it is covered enough?